ui/apps/platform/src/Containers/Policies/Wizard/Step1/PolicyCategoriesSelectField.tsx

```diff
@@ -60,12 +76,14 @@ function PolicyCategoriesSelectField(): ReactElement {
                 selections={field.value}
                 onSelect={onSelectHandler(field.value)}
                 onToggle={onCategoriesToggle}
-                onCreateOption={onCreateCategory}
+                onCreateOption={
+                    isPolicyCategoriesEnabled ? () => undefined as void : onCreateCategory
+                }
                 onClear={clearSelection}
-                isCreatable
+                isCreatable={!isPolicyCategoriesEnabled}
             >
-                {policyCategories.map((category) => (
-                    <SelectOption key={category} value={category} />
+                {policyCategories.map(({ id, name }) => (
+                    <SelectOption key={id} value={name} />
```

Comment:
I think you might have to conditionally do the `key={category} value={category}` mapping here based off of the feature flag